tests/records/full-record.json

```diff
@@ -59,8 +55,7 @@
     "title": "InvenioRDM",
     "additional_titles": [{
       "title": "a research data management platform",
-      "type": "subtitle",
-      "lang": "en"
```

Comment:
Minor: We have to add back the ``lang`` attribute on titles and descriptions. Sorry for the change of mind.

Reply:
Added back. What is the reasoning behind?